packages/network-graph/components/network-graph/sampleData.ts

```diff
@@ -0,0 +1,370 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const MiserableData = {
```

Comment:
wonder if we can move this sample data file to within the `__stories__` folder instead since it's not part of the component

Reply:
Done.